### DISCUSSION TITLE
Decide to what to do for 1 night, 2021-06-21 영종도 을왕리 해수욕장

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#43] [Workshop]
### Place

을왕리 The Week&Resort 입니다.

### 25 June

- 14:00 공항철도 청라국제도시역 파리바게트에서 만나서 출발
- 간식 필요한 것 추천, 없으면 역 안에 있는 파리바게트 + GS25에서 즉흥적으로 떙기는 것 구매
- 저녁 메뉴: 조개구이 4인 세트
- 눈치봐서 누군가 불꽃놀이 한다 싶으면 [깨진 유리창 이론](https://ko.wikipedia.org/wiki/%EA%B9%A8%EC%A7%84_%EC%9C%A0%EB%A6%AC%EC%B0%BD_%EC%9D%B4%EB%A1%A0)에 따라 우리도 합류해서 불꽃놀이 즐기기
- 야식 필요한 것 추천
  - 을왕리 훌랄라참숯바베큐 두마리, 배부르면 한마리
  - 편의점에서 맥주나 기타 음료+먹을 것들 구매

### 26 June

- 조식
- 수영장 이용 여부? (X)
- 카페? 제가 수영장 입장 전에 미리 구매
- 점심은 황해칼국수
- 대략 2시 전후 마시랑 까페

**Comment by hi-hyein** [comment]
윤우님과 저는 방을 따로잡기로 결정했습니다 땅땅땅~!

**Reply by jongfeel** [reply]
저번 미팅때 그러기로 하고 예약 하면 알려달라고 했었어요 ㅋㅋ

**Reply by hi-hyein** [reply]
앗 그랬군요ㅋㅋㅋㅋㅋㅋㅋㅋㅋ 예약하면 알려드리겠슴다~

**Reply by hi-hyein** [reply]
윤우님과 저 예약완료했습니덩ㅎㅎㅎ

**Comment by jongfeel** [comment]
@hi-hyein @yoonucho @Joseph-Cha 

1주일 남았습니다.
글을 몇 번 수정해서 계획을 세워봤습니다.
의견 있으면 바로 수정하겠습니다.
